solidity/contracts/BurntPixArchives.sol

```diff
@@ -75,6 +76,13 @@ contract BurntPixArchives is LSP8CappedSupply {
             _setData(_LSP4_CREATORS_ARRAY_KEY, hex"00000000000000000000000000000001");
             _setData(0x114bd03b3a46d48759680d81ebb2b41400000000000000000000000000000000, abi.encodePacked(_creator));
             _setData(bytes32(abi.encodePacked(_LSP4_CREATORS_MAP_KEY_PREFIX, hex"0000", _creator)) , hex"24871b3d00000000000000000000000000000000");
+            // royalties
+            _setData(0xc0569ca6c9180acc2c3590f36330a36ae19015a19f4e85c28a7631e3317e6b9d, abi.encodePacked(
```

Comment:
can you share some info about this?

Reply:
yeah @Ledespana. standard specified here https://github.com/lukso-network/LIPs/pull/141/files#diff-2ed66096a197880d37b2a3c9128292bdce35717221634337fc1020eae105b0deR50
so we have:
1. interface for type of account that will be receiving royalties as bytes4
2. address of the account
3. % as basis points (5%)

the resulting value is `0xc0569ca6c9180acc2c3590f36330a36ae19015a19f4e85c28a7631e3317e6b9d`